treeherder/push_health/builds.py

```diff
@@ -15,3 +15,13 @@ def get_build_failures(push, parent_push=None):
         mark_failed_in_parent(failures, get_build_failures(parent_push))
 
     return failures
+
+
+def get_build_in_progress_count(push):
+    build_types = JobType.objects.filter(name__contains="build")
```

Comment:
I was comparing this refactor with what I have in my pr and I was wondering why you filter by 'build' name here but not in `get_build_failures`? Same for tests below.

Reply:
The reason was that build failures will be of `state=busted`.  My assumption (which used to be true, but who knows for sure if it still is) was that ONLY builds could be in the `busted` state.  But I needed to use the filter since the state would be in-progress of some type.  But wouldn't hurt to do it in both places.  And maybe using `name__icontains` would be slightly safer, as mentioned in your PR.